exec/java-exec/src/main/resources/rest/login.ftl

```diff
@@ -24,15 +24,15 @@
 <#macro page_body>
     <div class="page-header"></div>
     <div align="center" class="table-responsive">
-        <form role="form" name="input" action="/j_security_check" method="POST">
+        <form role="form" name="input" autocomplete="off" action="/j_security_check" method="POST">
```

Comment:
Please revert back `autocomplete="off"`

Reply:
Due to a privacy concern, I would prefer that the browser not remember the values for such fields that may contain sensitive information (for example, a unique government identifier or credit card security code), even if the browser's autocomplete feature is enabled. 
Hence, autocomplete="off"

Reply:
Done, reverted autocomplete="off" as it is not fully supported by browsers. Please review.